<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTableW</class>
 <widget class="QMainWindow" name="QtTableW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTableW</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitterH">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="btnHheader">
          <property name="text">
           <string>设置水平表头</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnSetRowCount">
          <property name="text">
           <string>设置行数</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinRowCount">
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="btnInitData">
          <property name="text">
           <string>初始化表格数据</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="btnInsertRow">
          <property name="text">
           <string>插入行</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnAddRow">
          <property name="text">
           <string>添加行</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="btnDeleteRow">
          <property name="text">
           <string>删除当前行</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="btnAutoHeight">
          <property name="text">
           <string>自调节行高</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="btnAutoWidth">
          <property name="text">
           <string>自调节列宽</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btnReadData">
          <property name="text">
           <string>读取表格内容到文本</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="chkEditable">
          <property name="text">
           <string>表格可编辑</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="chkBgColor">
          <property name="text">
           <string>间隔行底色</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="chkHheader">
          <property name="text">
           <string>水平表头</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="chkVHeader">
          <property name="text">
           <string>垂直表头</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QRadioButton" name="radioSelRow">
          <property name="text">
           <string>行选择</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QRadioButton" name="radioSelCell">
          <property name="text">
           <string>单元格选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitterV">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tableWidget"/>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
